### ISSUE TITLE
OmniApp vs Server side java

### ISSUE THREAD

**Issue by punit-porwal** [OmniDB/OmniDB#1161]
Hi Team,

Not a issue or but, need enquiry 
Is it possible if we can use OmniApps(Front-End) as it is as you created for UI and Server side can we use Java based server code means we can install OmniApps in local and use own java based server code to make connection between App and any database , or server and Front-End both are tied with Python language only hence not able to do separation , could you please review and share you comments

Thanks
Punit

**Comment by luss** [comment]
Yes...   Your choice to use OmniDB as your RDBMS Admin front end does NOT
block you from getting at your any of your RDBMS databases from the
platform & tooling of your choice.  Long live open source & SQL!!

--Luss
